### PR TITLE
Whats supported topic

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/service-reliability-management/shared/supported.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Harness supports the following Health Sources and Change Sources.
```
Comment (roshnisarangadharan) [inline docs/service-reliability-management/shared/supported.md:1]: Sources are not capitalised in the following sub headers. Please check what is the correct usage.

File: docs/service-reliability-management/shared/supported.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Health sources
+
+A Health Source monitors changes in health trends of the Service using metrics and logs collected from an APM and log provider respectively.
```
Comment (roshnisarangadharan) [inline docs/service-reliability-management/shared/supported.md:5]: A health source monitors changes in health trends of the service using metrics and logs collected from an APM and log provider respectively.

File: docs/service-reliability-management/srm-whats-supported.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (roshnisarangadharan) [inline docs/service-reliability-management/srm-whats-supported.md:1]: same comments for this topic

**Review by roshnisarangadharan — DISMISSED** [review]
Approved with some comments. Implement as you see fit.

File: docs/service-reliability-management/srm-whats-supported.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+title: What's supported?
+description: Health and change sources supported by Harness SRM
+sidebar_position: 10
+---
+
+Harness supports the following health and change Sources:
```
Comment (roshnisarangadharan) [inline docs/service-reliability-management/srm-whats-supported.md:7]: sources

File: docs/service-reliability-management/srm-whats-supported.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## Health sources
+
+A Health Source monitors changes in health trends of the service using metrics and logs collected from Application Performance Monitoring (APM) or log providers respectively.
```
Comment (roshnisarangadharan) [inline docs/service-reliability-management/srm-whats-supported.md:11]: health source

File: docs/service-reliability-management/srm-whats-supported.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- New Relic
+- Prometheus
+- Splunk
+- Sumo Logic
+
+Harness also supports customized APMs. To know more about using customized APMs of your choice, go to [Custom Health Source](../continuous-delivery/verify/configure-cv/verify-deployments-with-custom-health-metrics.md).
+
+## Change sources
+
+A Change Source monitors change events related to deployments, infrastructure changes, and incidents.
```
Comment (roshnisarangadharan) [inline docs/service-reliability-management/srm-whats-supported.md:30]: change source

Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-whats-supported.md]: Udpated

File: docs/service-reliability-management/srm-whats-supported.md (hunk 5 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-whats-supported.md]: Updated

File: docs/service-reliability-management/shared/supported.md (hunk 1 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/shared/supported.md:1]: Updated

File: docs/service-reliability-management/shared/supported.md (hunk 2 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/shared/supported.md:5]: Updated

File: docs/service-reliability-management/shared/supported.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+- Custom Health Source
+- Google Cloud Operations (formerly Stackdriver)
+- New Relic
+- Datadog
+
+More tools will be added soon.
+
+### Change sources
+
+A Change Source monitors change events related to deployments, infrastructure changes, and incidents. The following Change Sources are supported:
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/service-reliability-management/shared/supported.md:26]: Updated

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**